locations/spiders/devernois.py

```
@@ -0,0 +1,16 @@
+from typing import Iterable
+
+from locations.categories import Categories, apply_category
+from locations.items import Feature
+from locations.storefinders.stockist import StockistSpider
+
+
+class Devernois(StockistSpider):
+    name = "devernois"
+    item_attributes = {"brand": "Devernois", "brand_wikidata": ""}
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)